docs/source/examples/plots_examples.rst

```diff
@@ -1,6 +1,182 @@
 Plotting with manim
 =================================
 
+Examples to illustrate the use of GraphScenes in manim
+
+
+.. manim:: Plot1
+    :quality: medium
+    :save_last_frame:
+
```

Comment:
**Nitpick :** Maybe add `import numpy as np` 

(here and in every example that uses numpy)

Reply:
When doing `from manim import *`, numpy is also imported as `np`. At some point, we have decided not to show imports in the examples -- but we can certainly discuss this again.

Reply:
Is numpy in __init__ ? 
But still, I think for good practice purpose we should add it.

Reply:
We can write at one point before the examples begin, that all examples are run with `from manim import *``and that this also includes `import numpy as np`

Reply:
I kinda like examples that are fully copy&pastable. I think in the best case scenario, any user would be able to copy the example code and run it immediately in their system. So I'm in favor of explictly including the import statments.

Reply:
I prefer it to have the code as compact as possible so that you don't have to scroll extra long.
In this case, we only have 9 examples, that are with the import statement 18 extra lines of code.
But maybe we can discuss this in a separate issue.
Can we leave it for now without the import statement?
When it's consensus that they should be included, then I am ok with changing it in a future pr.

Reply:
If the code is too long, then maybe each example should have its own page, like matplotlib does.

But yes let's leave that for another issue. I'm fine with merging this PR as is.

Reply:
nice!! 
See this question in of import statement in #454

Reply:
>  I'm fine with merging this PR as is.
@leotrs : could you then give it an approve? :)